README.md

```diff
@@ -1,204 +1,270 @@
-# There is a package that is not in PyPI
-pip install -r ./requirements/requirements.txt
-python setup.py install
+__NOTE THAT__ to exit the _virtual environment_ we use the `deactivate` command,
+as we can see below:
+
+```bash
+# When you no longer want to run the History service,
+# you need to leave the virtual environment created for it:
+$> deactivate ./venv
 ```
 
-The last command will generate a .egg file and install it into your virtual
-environment.
+To install all dependencies, execute the following commands.
 
-#### Docker Setup
+```bash
+# Some libraries depend on the specific version of the pip:
+$> python -m pip install pip==19.0.3
 
-Another alternative is to use **docker** to run the service. To build the
-container, from the repository's root:
+$> pip install versiontools
 
-```shell
+$> pip install -r ./requirements/requirements.txt
+```
+
+The next and last command will generate a
+[`.egg`](http://peak.telecommunity.com/DevCenter/PythonEggs) file and install
+it into your virtual environment:
+~~~bash
+$> python setup.py install
+~~~
+
+#### **History service on Docker**
+
+Another alternative is to use [Docker](https://www.docker.com/)  to run the
+service. To build the container, from the repository's root:
+
+```bash
 # you may need sudo on your machine:
 # https://docs.docker.com/engine/installation/linux/linux-postinstall/
-docker build -t <tag-for-history-docker> -f docker/history.docker .
+$> docker build -t <tag-for-history-docker> -f docker/history.docker .
 ```
-### How to run the service
 
-To run the standalone history service, just set all needed environment variables and execute:
+### **How to run the History service**
+
+To run the _History_ service on standalone mode, just set all needed environment
+variables and execute:
 
 ```bash
-./docker/falcon-docker-entrypoint.sh start
+# from the repository's root:
+$> ./docker/falcon-docker-entrypoint.sh start
 ```
 
 To run the service on docker, you can use:
-```shell
+```bash
 # may need sudo to run
-docker run -i -t <tag-for-history-docker>
+$> docker run -i -t <tag-for-history-docker>
 ```
-### API
-
-The API documentation for this service is written as API blueprints.
-To generate a simple web page from it, one may run the commands below.
-
-```shell
-npm install -g aglio # you may need sudo for this
 
-# static webpage
-aglio -i docs/history.apib -o docs/history.html
+********************************************************************************
 
-# serve apis locally
-aglio -i docs/history.apib -s
-```
+## **About Persister service**
 
-### Tests
+The _Persister_, as the name suggests, is the responsible for the persistence of
+the data sent from devices. It is always aware of Kafka's notifications, when
+new device data is registered in Kafka, _Persister_ receives a notification and
+starts the process of transferring data _from_ Kafka _to_ MongoDB. In this way,
+it is what populates the measurement history database of the devices.
```

Comment:
Suggestion:

The History service is only capable of retrieving data from the MongoDB instance, it is not able to store any data. It is the Persister's duty to make this happen: it is notified whenever a new message has arrived in Kafka and stores it in MongoDB.